index.js

```diff
@@ -250,12 +272,16 @@ DB.prototype.get = function (key, opts, cb) {
   }
   this.kv.get(key, opts, function (err, docs) {
```

Comment:
I think we're safe even if we don't return raw values. The API says that `opts` is passed into hyperkv, but makes no claims that what you got back from osm-p2p-db would be the same as whatever hyperkv would've returned.